README.md

```diff
@@ -187,12 +187,13 @@ When updating this, also update:
 
 Tokio will keep a rolling MSRV (minimum supported rust version) policy of **at
 least** 6 months. When increasing the MSRV, the new Rust version must have been
```

Comment:
It seems that tokio 1.27 (which bumps tokio MSRV to 1.56.0) is released in March, so according to this policy this PR should be delayed until September.

Reply:
That's not what this number means. It refers to the age of the rustc version being upgraded to.

Reply:
Thanks for clarification!